test/authority_test.go

```diff
@@ -98,7 +106,44 @@ func (s) TestUnix(t *testing.T) {
 	}
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
-			runUnixTest(t, test.address, test.target, test.authority)
+			runUnixTest(t, test.address, test.target, test.authority, nil)
+		})
+	}
+}
+
+// TestUnixCustomDialer does end to end tests with various supported unix target
+// formats, ensuring that the target sent to the dialer does NOT have the
+// "unix:" prefix stripped.
+func (s) TestUnixCustomDialer(t *testing.T) {
+	tests := []struct {
+		name      string
+		address   string
+		target    string
+		authority string
+	}{
+		{
+			name:      "Unix1",
```

Comment:
"Relative path" (and "Absolute path" below).  Can you change the ones above to make them similarly descriptive, too, please?

Reply:
Made this change.